docs/3_finetuner_in_action/3_2_image_to_image.md

```diff
@@ -1,3 +1,154 @@
+┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
+┃                                            model ┃           task ┃ output_dim ┃ architecture ┃                                          description ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
+│                                         resnet50 │ image-to-image │       2048 │          CNN │                               Pretrained on ImageNet │
+│                                        resnet152 │ image-to-image │       2048 │          CNN │                               Pretrained on ImageNet │
+│                                  efficientnet_b0 │ image-to-image │       1280 │          CNN │                               Pretrained on ImageNet │
+│                                  efficientnet_b4 │ image-to-image │       1280 │          CNN │                               Pretrained on ImageNet │
+│                     openai/clip-vit-base-patch32 │  text-to-image │        768 │  transformer │ Pretrained on millions of text image pairs by OpenAI │
+│                                  bert-base-cased │   text-to-text │        768 │  transformer │       Pretrained on BookCorpus and English Wikipedia │
+│ sentence-transformers/msmarco-distilbert-base-v3 │   text-to-text │        768 │  transformer │           Pretrained on Bert, fine-tuned on MS Marco │
+└──────────────────────────────────────────────────┴────────────────┴────────────┴──────────────┴──────────────────────────────────────────────────────┘
+```
+
+For this example, we're gonna go with `resnet50`.
+
+## Creating a fine-tuning job
+You can easily start a fine-tuning run with `finetuner.fit`. With Finetuner you can create several fine-tuning jobs which will run in parallel.
+Let's use this advantage now and create two runs with different `learning_rate` values.
+
+```python
+from finetuner.callback import BestModelCheckpoint, EvaluationCallback
+
+run1 = finetuner.fit(
+        model='resnet50',
+        run_name='resnet-ttl-1',
+        description='fine-tune the whole model.',
+        train_data='resnet-ttl-train-data',
+        eval_data='resnet-ttl-eval-data',
+        loss='TripletMarginLoss',
+        callbacks=[BestModelCheckpoint(), EvaluationCallback(query_data='resnet-ttl-eval-data')],
+        epochs=6,
+        learning_rate=0.001,
+    )
+
+run2 = finetuner.fit(
+        model='resnet50',
+        run_name='resnet-ttl-2',
+        description='freeze all weights, and only fine-tune the additional MLP layer.',
+        train_data='resnet-ttl-train-data',
+        eval_data='resnet-ttl-eval-data',
+        loss='TripletMarginLoss',
+        callbacks=[BestModelCheckpoint(), EvaluationCallback(query_data='resnet-ttl-eval-data')],
+        epochs=6,
+        learning_rate=0.01,
+    )
```

Comment:
when creating 2nd runs, please use a better set of configurations, learning rate `0.01` seems not quite reasonable to me